components/molecules/cards/card-project/adapters/default/default.adapter.tsx

```diff
@@ -22,49 +23,84 @@ export function CardProjectDefaultAdapter<C extends ElementType = "div">({
   bottomTags = [],
   primaryActionProps,
   secondaryActionProps,
+  maxBottomTags,
   htmlProps,
 }: CardProjectPort<C>) {
+  const Component = as || "div";
   const slots = CardProjectDefaultVariants();
 
+  const nbBottomTags = useMemo(() => bottomTags?.length ?? 0, [bottomTags]);
+  const isMaxBottomTags = useMemo(() => nbBottomTags > (maxBottomTags ?? 0), [nbBottomTags, maxBottomTags]);
```

Comment:
```suggestion
  const reachedMaxBottomTags = useMemo(() => nbBottomTags > (maxBottomTags ?? 0), [nbBottomTags, maxBottomTags]);
```

There is a problem with your condition, if no maxBottomTags is defined you will always consider the max has been reached.